website/package.json

```diff
@@ -41,10 +44,11 @@
     "prop-types": "15.7.2",
     "query-string": "6.14.1",
     "re-resizable": "6.9.0",
-    "react": "16.14.0",
+    "react": "17.0.1",
     "react-copy-to-clipboard": "5.0.3",
-    "react-dom": "16.14.0",
+    "react-dom": "17.0.1",
     "react-frame-component": "4.1.3",
+    "react-helmet": "6.1.0",
```

Comment:
How come we didn't need it before?

Reply:
Seems like react-helmet-async is sufficient . Will remove this.